Python/other/Program_to_rotation_array.py

```diff
@@ -0,0 +1,16 @@
+def array_rotation(): 
+    save = []
+    rest_arr = []
+    for i in arr:
+        if i == d or i < d:
+            save.append(i)
+        else:
+            rest_arr.append(i)
+
+    rest_arr.extend(save)
+    print('rest arry =', rest_arr)
+
+arr = list(input().split())
+d = input()
+n = input()
+array_rotation()
```

Comment:
At the end write Time & Space Complexity of the given Algorithm
And also write Sample Input and output in commented form

Reply:
As you requested I changes made please check